.github/workflows/release.yml

```diff
@@ -1,6 +1,7 @@
 name: Release
 
 on:
+  workflow_dispatch:
```

Comment:
can users outside of our org manually trigger jobs?

Reply:
from looking around, it looks like you need write access to the repo at least https://github.com/orgs/community/discussions/26053.

that's a good point though, should we only allow tags to trigger the release or allow manually triggering it?

Reply:
Manual triggers are great to retry stuff if there are failures.

Reply:
wdyt @gracedo?

Reply:
Sg if limited to users with write access